### DISCUSSION TITLE
Claim blocks per world

### DISCUSSION THREAD

**Discussion by Flying--Dutchman** [GriefPrevention/GriefPrevention#628] [Ideas]
As a administrator, I would like to see an option to gain and spend claim blocks per world.
I have a creative, and a survival world. ATM players that spend a lot of time in creative get a lot of claim blocks easy. They than switch to the survival world to spend those claim blocks.

This new feature would make sure that time spend in one world, won't get you claim blocks to use in a different world.

**Comment by bjorno43** [comment]
I also would love this idea for the exact same reasons. I use PerWorldInventories and Multiverse Core to make sure players can enjoy both survival and creative modes without being able to cheat traveling between them.

I wouldn't even mind to just be able to only set a static amount of claim blocks for creative worlds. I mean, they're playing creative. I don't see why players should "earn" claim blocks in this mode. They can't really do much in the world except within their own claims anyway.
